build.gradle

```diff
@@ -183,6 +184,7 @@ apply from: file('gradle/hacks/solr.findbugs.gradle')
 apply from: file('gradle/hacks/wipe-temp.gradle')
 apply from: file('gradle/hacks/hashmapAssertions.gradle')
 apply from: file('gradle/hacks/turbocharge-jvm-opts.gradle')
+apply from: file('gradle/hacks/dummy-outputs.gradle')
```

Comment:
can we call this "placeholder-outputs" instead?

Reply:
I'd rather keep it consistent with Lucene, if you don't mind. Not a native speaker but is something wrong with the "dummy"? I am aware of other senses but this literal definition suits me just fine in this context "an object designed to resemble and serve as a substitute for the real or usual one."?